var/spack/repos/builtin/packages/py-userpath/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyUserpath(PythonPackage):
+    """Cross-platform tool for adding locations to the user PATH."""
+
+    homepage = "https://github.com/ofek/userpath"
+    pypi = "userpath/userpath-1.8.0.tar.gz"
+
+    version("1.8.0", sha256="04233d2fcfe5cff911c1e4fb7189755640e1524ff87a4b82ab9d6b875fee5787")
+
+    depends_on("python@3.7:", type=("build", "run"))
+
+    depends_on("py-hatchling@1.17.0:", type="build")
+
+    depends_on("py-click@8.1.3:", type=("build", "run"))
```

Comment:
Where do you see these version requirements?

Reply:
The package itself doesn't seem to specify any version requirements, so to reduce surprises I marked them with the current latest versions in `develop`. I can remove the requirements if you'd prefer to match upstream.

Reply:
Let's remove the requirements to match upstream

Reply:
Done.